app/views/components/strong.scala.html

```diff
@@ -16,6 +16,6 @@
 
 @this()
 
-@(content: Html)(implicit messages: Messages)
+@(content: String, withFullStop: Boolean = false)(implicit messages: Messages)
```

Comment:
Does changing this to a String break anything else? Are there any views that were passing HTML into this block? - I guess it wouldn't compile if that was the case so should be fine tbf.

If the `withFullStop` was added due to there being a space in the view when rendered, then that could have been solved by just removing any whitespace/carriage returns from the end of this component - I think Lewis did that one one of his PRs, but I'm not sure if that's been merged yet or not.